src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandler.java

```diff
@@ -57,4 +61,12 @@ protected void updateCaseStatus(SscsCaseData caseData) {
             caseData.setInterlocReviewState(InterlocReviewState.AWAITING_ADMIN_ACTION);
         }
     }
+
+    private DocumentType getPostHearingReviewDocumentType(PostHearing postHearing) {
```

Comment:
Could probably stick this in SscsUtil, I did have something similar in a PR for an on-hold ticket (https://github.com/hmcts/sscs-tribunals-case-api/pull/3218/files#diff-16f663f954ea900ea51d362e5ba5cc505f93042d86fcef54605103d3c357ace8), might be worth cribbing from there for your own purposes.